pkg/cmd/dev/test.go

```diff
@@ -257,6 +257,26 @@ func (d *dev) test(cmd *cobra.Command, commandLine []string) error {
 		testTargets = append(testTargets, target)
 	}
 
+	// List of bazel integration tests that will fail when running `dev test pkg/...`
+	var integrationTests = map[string][2]string{
```

Comment:
Let's move the declaration of this variable out of this function and make it global. It's a constant map, it doesn't need to be here in the middle of the function.